banners/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+
+from banners.models import Banner
+
+
+class BannerAdmin(admin.ModelAdmin):
+    list_display = ("title", "active", "psf_pages_only")
+
+admin.site.register(Banner, BannerAdmin)
```

Comment:
Style nit: We are trying to use the `@admin.site.register()` decorator version in new code.